src/components/interlocutor/reducers/index.js

```diff
@@ -1,22 +1,33 @@
-import { ADD_INTERLOCUTOR } from '../screens/InterlocutorAdd/actions/types';
-
 // To be deleted when activity add is working
+
 import moment from 'moment';
 import 'moment/locale/fr';
+
+import { ADD_INTERLOCUTOR } from '../screens/InterlocutorAdd/actions/types';
+import { GET_INTERLOCUTOR_LIST } from '../screens/InterlocutorList/actions/types';
+import { GET_INTERLOCUTOR_ACTIVITY } from '../screens/InterlocutorDisplay/actions/types';
```

Comment:
I think there is some naming inconsistency here : for the elements `activityList` and `interlocutorList` (which are both lists), we have the names `GET_INTERLOCUTOR_LIST` and `GET_INTERLOCUTOR_ACTIVITY`. Should rename the second one to `GET_INTERLOCUTOR_ACTIVITY_LIST`